ansible/roles/candlepin/README.md

```diff
@@ -13,14 +13,6 @@ Candlepin has many requirements for its developers. You will need a Fedora or Ce
 enabled to use this role. It is recommmended that that box has a non-root user to run Candlepin on (see
 variables below).
 
-Dependencies
-------------
-
-This role depends on the `rvm1.ruby` role for installing and configuring RVM. At the time of writing, Candlpin
```

Comment:
we also need to remove the reference of the role from `ansible/requirements.yml`